### PR TITLE
Fix permission bug for Android 6.0

### PR DESCRIPTION
**Pull request by CoXier**
Sample has not supported Android M yet.

### CODE REVIEW

File: picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java [hunk 1]
```diff
@@ -30,6 +34,9 @@
     implements LoaderManager.LoaderCallbacks<Cursor> {
   private SampleContactsAdapter adapter;
 
+  private static final String sReadContacts = Manifest.permission.READ_CONTACTS;
```
Comment (FabianTerhorst) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: sReadContacts -> READ_CONTACTS

File: picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java [hunk 2]
```diff
@@ -30,6 +34,9 @@
...
+  private static final int sReadContactsCode = 100;
```
Comment (FabianTerhorst) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: sReadContactsCode -> READ_CONTACTS_CODE

Comment (CoXier) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: Is it picasso's style ? It's different from [AOSP](https://source.android.com/source/code-style).

Ok I will change it.

Comment (FabianTerhorst) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: Constants are written in caps.

Comment (JakeWharton) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: AOSP's style is a historical accident and shouldn't be used anywhere except inside AOSP.

Comment (CoXier) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: Ok I get it.

Comment (CoXier) [inline picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java]: Is there anything for this pr to do?

**Comment by CoXier** [comment]
Although this is a bug but it does not matter with this lib.

**Pull request closed without merging**